src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    width: 160px;
+    height: 134px;
+  }
```

Comment:
The padding property on line 8 seems to be an unusual value. It's best to stick to a consistent set of spacing values in your design. This could be a potential design issue.